mmpose/core/keypoint/heatmap.py

```diff
@@ -1,719 +0,0 @@
-        - keypoint_weights (np.ndarray): The target weights in shape (K,)
-
-    .. _`The Devil is in the Details: Delving into Unbiased Data Processing for
-    Human Pose Estimation`: https://arxiv.org/abs/1911.07524
-    """
-
-    num_keypoints = keypoints.shape[0]
-    image_size = np.array(image_size)
-    w, h = heatmap_size
-    feat_stride = (image_size - 1) / [w - 1, h - 1]
-    keypoint_weights = np.ones(num_keypoints, dtype=np.float32)
-
-    if not combined_map:
-        heatmap = np.zeros((num_keypoints, h, w), dtype=np.float32)
-
-        # 3-sigma rule
-        radius = factor * 3
-
-        # xy grid
-        gaussian_size = 2 * radius + 1
-        x = np.arange(0, gaussian_size, 1, dtype=np.float32)
-        y = x[:, None]
-
-        for i in range(num_keypoints):
-            # skip unlabled keypoints
-            if keypoints_visible[i] < 0.5:
-                keypoint_weights[i] = 0
-                continue
-
-            mu = (keypoints[i] / feat_stride + 0.5).astype(np.int64)
-            # check that the gaussian has in-bounds part
-            left, top = (mu - radius).astype(np.int64)
-            right, bottom = (mu + radius + 1).astype(np.int64)
-
-            if left >= w or top >= h or right < 0 or bottom < 0:
-                keypoint_weights[i] = 0
-                continue
-
-            mu_ac = keypoints[i] / feat_stride
-            x0 = y0 = gaussian_size // 2
-            x0 += mu_ac[0] - mu[0]
-            y0 += mu_ac[1] - mu[1]
-            gaussian = np.exp(-((x - x0)**2 + (y - y0)**2) / (2 * factor**2))
-
-            # valid range in gaussian
-            g_x1 = max(0, -left)
-            g_x2 = min(w, right) - left
-            g_y1 = max(0, -top)
-            g_y2 = min(h, bottom) - top
-
-            # valid range in heatmap
-            h_x1 = max(0, left)
-            h_x2 = min(w, right)
-            h_y1 = max(0, top)
-            h_y2 = min(h, bottom)
-
-            heatmap[i, h_y1:h_y2, h_x1:h_x2] = gaussian[g_y1:g_y2, g_x1:g_x2]
-
-    else:
-        heatmap = np.zeros((num_keypoints, 3, h, w), dtype=np.float32)
-        x = np.arange(0, w, 1)
-        y = np.arange(0, h, 1)[:, None]
-
-        # positive area radius in the classification map
-        radius = factor * max(w, h)
-
-        for i in range(num_keypoints):
-            if keypoints_visible[i] < 0.5:
-                keypoint_weights[i] = 0
-                continue
-
-            mu = keypoints[i] / feat_stride
-
-            x_offset = (mu[0] - x) / radius
-            y_offset = (mu[1] - y) / radius
-
-            heatmap[i, 0] = np.where(x_offset**2 + y_offset**2 <= 1, 1., 0.)
-            heatmap[i, 1] = x_offset
-            heatmap[i, 2] = y_offset
-
-        # keep only valid region in offset maps
-        heatmap[:, 1:] *= heatmap[:, :1]
-        heatmap = heatmap.reshape(num_keypoints * 3, h, w)
-
-    return heatmap, keypoint_weights
-
-
-def get_max_preds(heatmaps: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
Please replace the places using this function `get_max_preds`, for example, with `get_heatmap_maximum` in `mmpose/mmpose/core/keypoint/codecs/utils.py`